server/src/domain/person/person.service.ts

```diff
@@ -147,6 +148,56 @@ export class PersonService {
     return true;
   }
 
+  async unMergePerson(authUser: AuthUserDto, dto: UnMergePersonDto): Promise<BulkIdResponseDto> {
+    await this.access.requirePermission(authUser, Permission.PERSON_WRITE, dto.personId);
+
+    const oldPerson = await this.findOrFail(dto.personId);
+
+    const newPerson = await this.repository.create({ ownerId: authUser.id });
+    let result: BulkIdResponseDto;
+    try {
+      const mergePerson = await this.repository.getById(oldPerson.id);
+      if (!mergePerson) {
+        result = { id: oldPerson.id, success: false, error: BulkIdErrorReason.NOT_FOUND };
+      }
+
+      this.logger.log(`un-merging ${dto.assetId} from ${oldPerson.id}`);
+      await this.repository.removeFaceFromPerson(oldPerson.id, newPerson.id, dto.assetId);
+
+      await this.repository.update({
+        id: newPerson.id,
+        faceAssetId: dto.assetId,
+      });
+
+      const assetId = dto.assetId;
+      const face = await this.repository.getFaceById({ personId: newPerson.id, assetId });
```

Comment:
You should look up the face right after the person and throw an error before doing anything if it is invalid.